### ISSUE TITLE
bug: Missing Linux launch related Spanish localizations

### ISSUE THREAD

**Issue by 0neGal** [0neGal/viper#242]
It seems I managed to miss this when merging #229, but the Spanish localizations only has 2 of the titles for the new toasts instead of all of them + the text... This is where I recommend using `npm run langs:localize` over manually editing :p

~~Though, I suppose this is my fault for never getting around to making some CI/CD for checking localizations...~~ This is now added with [5ddec75](https://github.com/0neGal/viper/commit/5ddec759a2a71c553cfffbab02033fdb37470a2d)

Specifically these are missing (@XNovaDelta):
```
es.json is missing:
  gui.toast.title.missing_steam
  gui.toast.title.missing_flatpak
  gui.toast.title.missing_flatpak_steam
  gui.toast.desc.failed_launch_command
  gui.toast.desc.missing_launch_command
  gui.toast.desc.missing_steam
  gui.toast.desc.missing_flatpak
  gui.toast.desc.missing_flatpak_steam
```

**Comment by 0neGal** [comment]
Well, I got curious, and discovered I accidentally broke the checking of the localization files when publishing a build ([ccde19d8](https://github.com/0neGal/viper/commit/ccde19d82a1a8fd3c5ea20efa31a0e18393e14fe)), the entire reason why I never noticed this when releasing, that issue is fixed now: [877d0e5](https://github.com/0neGal/viper/commit/877d0e5d50712fb0a3ee915d3bbf6991c39f8b7c)

whoopsie

**Comment by 0neGal** [comment]
Superseded by #256
